### ISSUE TITLE
bug: Cannot create schedule (error returned from database: null value in column "enabled" of relation "schedule")

### ISSUE THREAD

**Issue by jaller94** [windmill-labs/windmill#392]
## Describe the bug
A database error is preventing me from creating any schedules.

## To Reproduce
1. Go to https://app.windmill.dev/schedule/add
2. Configure your schedule.
3. Click on "Schedule"

## Expected behavior
A schedule gets created.

## Screenshots
```
POST https://app.windmill.dev/api/w/j94_test/schedules/create
```
returns HTTP 500
```
Internal: inserting schedule in j94_test: error returned from database: null value in column "enabled" of relation "schedule" violates not-null constraint
```

**Windmill version**: v1.29.0-8-ga8594342

**Comment by rubenfiszel** [comment]
Should be fixed in 359d389

**Comment by rubenfiszel** [comment]
Fixed and deployed
